_docs/developer/troubleshooting/reinstallation_troubleshooting.md

```diff
@@ -0,0 +1,97 @@
+---
```

Comment:
this shouldn't be a separate page, its not troubleshooting, they are development instructions (less commonly used, but still development instructions)